### ISSUE TITLE
java.lang.UnsupportedOperationException occured when upgrading to v1.1.6

### ISSUE THREAD

**Issue by yuki-takei** [grails/grails-views#145]
Environment
--------------

### Before upgrade

* [Grails 3.2.7 with views-json:1.1.5](https://mvnrepository.com/artifact/org.grails/grails-bom/3.2.7)
* org.grails.plugins:mongodb:6.0.9

### After upgrade

* [Grails 3.2.8 with views-json:1.1.6](https://mvnrepository.com/artifact/org.grails/grails-bom/3.2.8)
* org.grails.plugins:mongodb:6.0.9

What happens?
----------------

1. `respond` an instance of domain class in a controller method (`jp.co.weseek.bevelle.api.WorkerController.getWorker`)
1. Following exception occured

    ```
    Caused by: grails.views.ViewRenderException: Error rendering view: MappingFactory not supported by implementation. Defined by Grails itself.
    	at grails.views.AbstractWritableScript.writeTo(AbstractWritableScript.groovy:43)
    	at grails.views.mvc.GenericGroovyTemplateView.renderMergedOutputModel(GenericGroovyTemplateView.groovy:73)
    	at org.springframework.web.servlet.view.AbstractView.render(AbstractView.java:303)
    	at grails.views.mvc.renderer.DefaultViewRenderer.render(DefaultViewRenderer.groovy:111)
    	at grails.artefact.controller.RestResponder$Trait$Helper.internalRespond(RestResponder.groovy:188)
    	at grails.artefact.controller.RestResponder$Trait$Helper.respond(RestResponder.groovy:98)
    	at jp.co.weseek.bevelle.api.WorkerController.getWorker(WorkerController.groovy:130)
    	... 63 common frames omitted
    Caused by: java.lang.UnsupportedOperationException: MappingFactory not supported by implementation. Defined by Grails itself.
    	at org.grails.datastore.gorm.config.GrailsDomainClassMappingContext.getMappingFactory(GrailsDomainClassMappingContext.java:60)
    	at grails.plugin.json.view.api.internal.DefaultGrailsJsonViewHelper.isSimpleValue(DefaultGrailsJsonViewHelper.groovy:518)
    	at grails.plugin.json.view.api.internal.DefaultGrailsJsonViewHelper.handleValue(DefaultGrailsJsonViewHelper.groovy:327)
    	at grails.plugin.json.view.api.internal.DefaultGrailsJsonViewHelper$_getIterableWritable_closure3.doCall(DefaultGrailsJsonViewHelper.groovy:267)
    	at grails.plugin.json.view.api.internal.DefaultGrailsJsonViewHelper$4.writeTo(DefaultGrailsJsonViewHelper.groovy:283)
    	at grails.plugin.json.view.JsonViewTemplate.json(JsonViewTemplate.groovy:126)
    	at bevelle_back_object__object_gson.run(bevelle_back_object__object_gson:5)
    	at grails.plugin.json.view.JsonViewTemplate.doWrite(JsonViewTemplate.groovy:35)
    	at grails.views.AbstractWritableScript.writeTo(AbstractWritableScript.groovy:40)
    	... 69 common frames omitted
    ```

### Before upgrade

* `jsonView.mappingContext` is null at [DefaultGrailsJsonViewHelper#516](https://github.com/grails/grails-views/blob/v1.1.5/json/src/main/groovy/grails/plugin/json/view/api/internal/DefaultGrailsJsonViewHelper.groovy#L516)
* [DefaultGrailsJsonViewHelper#521](https://github.com/grails/grails-views/blob/v1.1.5/json/src/main/groovy/grails/plugin/json/view/api/internal/DefaultGrailsJsonViewHelper.groovy#L521) is invoked

### After upgrade

* `jsonView.mappingContext` is **NOT** null at [DefaultGrailsJsonViewHelper#518](
https://github.com/grails/grails-views/blob/v1.1.6/json/src/main/groovy/grails/plugin/json/view/api/internal/DefaultGrailsJsonViewHelper.groovy#L518)
* [org.grails.datastore.gorm.config.GrailsDomainClassMappingContext#60](https://github.com/grails/grails-data-mapping/blob/v6.0.9/grails-datastore-gorm-support/src/main/groovy/org/grails/datastore/gorm/config/GrailsDomainClassMappingContext.java#L60) is invoked

Question
----------

* How to avoid this?

**Comment by jameskleeh** [comment]
Please upload an example application that demonstrates the issue

**Comment by yuki-takei** [comment]
@jameskleeh 
Here -> https://github.com/yuki-takei/test-grails-views-145
Any help would be appreciated.

**Comment by jameskleeh** [comment]
@yuki-takei Unfortunately this issue cannot be fixed within this library. It requires a change to GORM for Mongo. I've pushed a commit to resolve the issue there: https://github.com/grails/gorm-mongodb/commit/e4b9e96d1815c0ab76d48976938054348b535c29

The change will be available in the next 6.0.x and 6.1.x releases

**Comment by yuki-takei** [comment]
@jameskleeh Thank you! GJ!
